package.json

```diff
@@ -25,15 +25,19 @@
     "husky": "^7.0.0",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
-    "pg-format": "^1.0.4"
+    "pg-format": "^1.0.4",
```

Comment:
pg-format is arguably needed for test and dev side of this project so should be a normal dependency